core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -140,6 +140,19 @@ export function withEditor() {
                     const origin = params.clientHost || window.location.origin;
                     const iframeURL = new URL(pageAPIQueryParams, origin);
 
+                    const unit = device?.inode !== 'default' ? BASE_IFRAME_MEASURE_UNIT : '%';
+
+                    const wrapper =
```

Comment:
This can be a function? is a little difficult to read

Reply:
I like the idea